prosper0gdb/main.c

```diff
@@ -11,6 +11,22 @@
 #include <sys/uio.h>
 #include <sys/mount.h>
 
+void* dlsym(void*, const char*);
+
+void notify(const char* s)
+{
+    struct
+    {
+        char pad1[0x10];
+        int f1;
+        char pad2[0x19];
+        char msg[0xc03];
+    } notification = {.f1 = -1};
+    char* d = notification.msg;
+    while(*d++ = *s++);
+    ((void(*)())dlsym((void*)0x2001, "sceKernelSendNotificationRequest"))(0, &notification, 0xc30, 0);
+}
```

Comment:
Copy from ps5-kstuff/main.c, probably it needs to be refactored in some common place, but I didn't find good place to put it.